pytheus/registry.py

```diff
@@ -23,18 +27,20 @@ def collect(self) -> Iterable:
 class CollectorRegistry:
     def __init__(self, prefix: str = None) -> None:
         self._lock = Lock()
-        self._prefix = prefix
+        self.prefix = prefix
         self._collectors: dict[str, Collector] = {}
 
     def register(self, collector: Collector) -> None:
         with self._lock:
             if collector.name in self._collectors:
+                log.warning(f"collector with name '{collector.name}' already registred")
```

Comment:
ideally on log lines we shouldn't use f-strings, as they get compiled even if the level gets ignored , but it's ok 👍